Cargo.toml

```diff
@@ -15,9 +15,45 @@ edition = "2021"
 homepage = "https://aptoslabs.com"
```

Comment:
Shouldn't we change that too?

Reply:
Done (after quick discussion on Slack) 😄